code/modules/cargo/packs.dm

```diff
@@ -473,29 +473,43 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	crate_name = "Replacement lights"
 	group = "Engineering"
 
-/datum/supply_pack/metal50
-	name = "50 metal sheets"
+/datum/supply_pack/metal120
+	name = "120 metal sheets"
 	contains = list(/obj/item/stack/material/steel)
-	amount = 50
-	cost = 1000
+	amount = 120
+	cost = 360
 	containertype = /obj/structure/closet/crate/secure
 	crate_name = "Metal sheets crate"
 	group = "Engineering"
 
+
+/datum/supply_pack/metal480
+	name = "Bulk metal crate"
+	contains = list(/obj/item/stack/material/steel/full,
+	/obj/item/stack/material/steel/full,
+	/obj/item/stack/material/steel/full,
+	/obj/item/stack/material/steel/full)
+	cost = 1000
+	containertype = /obj/structure/largecrate
+	crate_name = "Bulk metal crate"
+	group = "Engineering"
+
+
+
 /datum/supply_pack/glass50
-	name = "50 glass sheets"
+	name = "120 glass sheets"
 	contains = list(/obj/item/stack/material/glass)
-	amount = 50
-	cost = 1000
+	amount = 120
+	cost = 360
 	containertype = /obj/structure/closet/crate/secure
 	crate_name = "Glass sheets crate"
 	group = "Engineering"
 
 /datum/supply_pack/wood50
-	name = "50 wooden planks"
+	name = "120 wooden planks"
 	contains = list(/obj/item/stack/material/wood)
-	amount = 50
-	cost = 1000
+	amount = 120
+	cost = 1200
```

Comment:
You may use defines for defining one sheet price and than make `cost = ammount * WOOD_SHEET_PRICE`, i think this would be better than multiplying these numbers with your hands

Reply:
I will do this later in future economy work, since there are other places it will need to be written in too